cfpb/cfgov-refresh
# Update django-college-costs-comparison to 1.6

A new version (`1.6.1`) of "Paying for College" (`django-college-costs-comparison`) has been released. Changes can be found [in this pull request](https://github.com/cfpb/django-college-costs-comparison/pull/11) and [this pull request](https://github.com/cfpb/django-college-costs-comparison/pull/12).

The changes to Paying for College here focus on the removal of Raphael, a package with known security issues. There are also minor fixes with regards to quotes and `rel="noopener noreferrer"` on some links.

Please note this is a security fix for an existing application. There are many known issues (see testing below) that this PR will not address.

## Changes
- Update optional-public.txt to use version 1.6.1 of the Paying for College repo, django-college-costs-comparison

## Testing
1. Download the `django-college-costs-comparison` repo.
2. From inside `cfgov-refresh`, install Paying for College with `pip install -e /path/to/django-college-costs-comparison/`
3. Check the url at [http://127.0.0.1:8000/paying-for-college/compare-financial-aid-and-college-cost/](http://127.0.0.1:8000/paying-for-college/compare-financial-aid-and-college-cost/)
4. Nothing should be visually or functionally different from the live site.


## Notes
- This update to Paying for College removes Raphael, but patches the application so that it has exactly the same functionality and appearance.
- The application (which relies on Javascript) does not have a graceful failure for no-js/IE8, but it didn't before. Adding one is outside the scope of this work.
- There is an unrelated bug in IE9 that I'll file a separate issue about.

## Checklist

## Testing checklist

### Browsers
- [ ] Internet Explorer 8, 9, 10, and 11 (Known issues in IE8 and IE9, see Notes above)

### Accessibility
- [ ] Keyboard friendly (Known issues, this PR will not fix. Accessibility audit forthcoming.)
- [ ] Screen reader friendly (Known issues, this PR will not fix. Accessibility audit forthcoming.)

### Other
- [ ] Is useable without JS (Known issue, this PR will not fix)
- [ ] Flexible from small to large screens (Known issues, this PR will not fix)
- [ ] No linting errors or warnings (Known issue, this PR will not fix)
- [ ] JavaScript tests are passing (Known issue, this PR will not fix)
